strawberry/django/views.py

```diff
@@ -23,6 +24,7 @@
     parse_request_data,
     process_result,
 )
+from strawberry.http.json_encoder import JsonEncoder, default_json_encoder
```

Comment:
I should probably change `JsonEncoder` to `JSONEncoder`, but note the name being the same as python's json.JSONEncoder (and DjangoJSONEncoder)